scripts/extension_helpers.js

```diff
@@ -3,22 +3,29 @@
 
 var path = require('path');
 var findImports = require('find-imports');
+var walkSync = require('walk-sync');
 
-// Get the list of vendor files.
+
+// Get the list of vendor files, with CodeMirror files being separate.
 var VENDOR_FILES = findImports('../lib/**/*.js', { flatten: true });
 var CODEMIRROR_FILES = VENDOR_FILES.filter(function(importPath) {
   return importPath.indexOf('codemirror') !== -1;
 });
-var codemirrorPaths = CODEMIRROR_FILES.map(function(importPath) {
-  return importPath.replace('.js', '');
-});
+CODEMIRROR_FILES.push('codemirror/lib/codemirror.js');
 VENDOR_FILES = VENDOR_FILES.filter(function (importPath) {
   return (importPath.indexOf('codemirror') !== 0 &&
           importPath.indexOf('phosphor') !== 0 &&
           importPath.indexOf('jupyter-js-services') !== 0);
 });
 
-/*
+// Get the list of codemirror imports.
+var codemirrorPaths = CODEMIRROR_FILES.map(function(importPath) {
+  return importPath.replace('.js', '');
```

Comment:
probably not a problem for codemirror, but this should only be deleting a trailing `.js`.